src/Test/Utilities/Portable/Compilation/OperationTreeVerifier.cs

```diff
@@ -44,7 +50,8 @@ public static void Verify(Compilation compilation, IOperation operation, string
 
         public static string GetOperationTree(Compilation compilation, IOperation operation, int initialIndent = 0)
         {
-            var walker = new OperationTreeVerifier(compilation, operation, initialIndent);
+            var walker = new OperationTreeVerifier(compilation, operation, initialIndent, 
+                trackExplicitNodes: operation.Language == LanguageNames.VisualBasic);
```

Comment:
I assume we'll apply the same invariant to IOperation in C# at some point. Is that correct?

Reply:
> I assume we'll apply the same invariant to IOperation in C# at some point. Is that correct?

Yes, the invariant is applicable to both languages.

Reply:
Is there a separate issue tracking enabling this for C# then?

Reply:
> Is there a separate issue tracking enabling this for C# then?

There is an existing issue about this - https://github.com/dotnet/roslyn/issues/22229. I am simply volunteering to do this work.